src/lib/parser.c

```diff
@@ -1831,8 +2026,9 @@ RdbStatus rdbLoadFloatValue(RdbParser *p, float *val) {
 }
 
 RdbStatus rdbLoadDoubleValue(RdbParser *p, double *val) {
-    BulkInfo *binfoUnused;
-    IF_NOT_OK_RETURN(rdbLoad(p, sizeof(*val), RQ_ALLOC_REF, (char *) val, &binfoUnused));
+    BulkInfo *binfo;
+    IF_NOT_OK_RETURN(rdbLoad(p, sizeof(*val), RQ_ALLOC, NULL, &binfo));
+    *val = *((double*) binfo->ref);
     memrev64ifbe(val);
```

Comment:
If you intend to push my fix, then please do it also for `rdbLoadFloatValue()`

Reply:
done